wordpress-mobile/WordPress-Android
# Revert work after editor v1.23 that landed in WPAndroid 14.4

A block editor crash when adding a button block has reached the main app and released on PlayStore as v14.4 of WPAndroid. This PR removes the issue by rolling back to the proper v1.23 editor release and will be released as hotfix v14.4.1 of WPAndroid.

To test:
1. Start a Gutenberg post
2. Add a "Button" block. The editor should work as normal and the block should render

PR submission checklist:
